packages/main/src/themes/AvatarGroup.css

```diff
@@ -55,31 +55,31 @@
 
 ::slotted([ui5-button]),
 :host([avatar-size="S"]) ::slotted([ui5-button]),
-:host([avatar-size="S"]) .ui5-avatar-group-overflow-btn {
+.ui5-avatar-group-overflow-btn.ui5-avatar-group-overflow-btn-s {
 	height: 3rem;
 	width: 3rem;
 	min-width: 3rem;
 	font-size: 1.125rem;
 }
 
 :host([avatar-size="M"]) ::slotted([ui5-button]),
```

Comment:
We still use the `avatar-size` attribute to style the slotted button (`overflowButton` slot), we could possibly add the size class to their parent container and style them both, like: 

```
.ui5-avatar-group-size-m ::slotted([ui5-button]),
.ui5-avatar-group-size-m .ui5-avatar-group-overflow-btn
```

Reply:
The selector for the attribute ```avatar-size``` is a leftover and it is not working anymore. It is deleted now.